Uplift/Views/Cells/FacilitiesDropdownCell.swift

```diff
@@ -150,33 +110,63 @@ extension FacilitiesDropdownCell: UICollectionViewDataSource {
     }
 
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
-        // TODO: once facility cells are finished
-        return UICollectionViewCell(frame: .zero)
+        switch facility.details[indexPath.row].detailType {
+        case .equipment:
+            //swiftlint:disable:next force_cast
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: Identifiers.facilityEquipmentListCell, for: indexPath) as! EquipmentListCell
+            cell.configure(for: facility.details[indexPath.row].getEquipmentCategories())
+            return cell
+        case .hours:
+            //swiftlint:disable:next force_cast
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: Identifiers.facilityHoursCell, for: indexPath) as! FacilitiesHoursCell
+            cell.configure(facilityDetail: facility.details[indexPath.row], dayIndex: 0, onChangeDay: nil)
+            return cell
+        case .prices:
+            //swiftlint:disable:next force_cast
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: Identifiers.facilitiesPriceInformationCell, for: indexPath) as! PriceInformationCell
+            cell.configure(items: facility.details[indexPath.row].items, prices: facility.details[indexPath.row].prices)
+            return cell
+        case .subfacilities:
+            //swiftlint:disable:next force_cast
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: Identifiers.facilitiesMiscellaneousCell, for: indexPath) as! MiscellaneousInfoCell
+            cell.configure(for: facility.details[indexPath.row].subfacilities)
+            return cell
+        }
+
     }
 
 }
 
 extension FacilitiesDropdownCell: UICollectionViewDelegate, UICollectionViewDelegateFlowLayout {
 
     func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
-        // TODO: once facility cells are finished
-        return .zero
+        let width = collectionView.bounds.width
+        let height = FacilitiesDropdownCell.getHeight(for: facility.details[indexPath.row])
+        return CGSize(width: width, height: height + FacilitiesDropdownCell.collectionViewSpacing)
     }
 
 }
 
 extension FacilitiesDropdownCell: DropdownViewDelegate {
 
     func dropdownViewClosed(sender dropdownView: DropdownView) {
-        // TODO: Implement
+        return
     }
```

Comment:
just change it to: `func dropdownViewClosed(sender dropdownView: DropdownView) {}`